test/CallbackTest.php

```diff
@@ -116,4 +117,22 @@ public function optionsCallback($value)
         $this->assertContains('something', $args);
         return $args;
     }
+
+    public function testIsValidWithNoCallback()
```

Comment:
Rename this to indicate what *behavior* you're testing: 'testIsValidRaisesExceptionInAbsenceOfCallback'. I can make that change on merge.